superset/assets/javascripts/components/ModalTrigger.jsx

```diff
@@ -15,6 +15,7 @@ const propTypes = {
   bsSize: PropTypes.string,
   className: PropTypes.string,
   tooltip: PropTypes.string,
+  disabled: PropTypes.bool,
```

Comment:
Oh right, I ended up backtracking on this approach when I realized I could disable my own `triggerNode` prop that I pass to it.